catalog/plugin/catalog-plugin-versioning/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -23,7 +23,7 @@
         <argument ref="catalogFramework" />
```

Comment:
We can also remove `xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs"` and associated schemaLocation references from this file